app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -413,17 +413,17 @@ export default {
       <a class="btn btn-large btn-primary btn-moon" @click="onHandleJoinCTA">Join Now</a>
     </div>
 
-    <div class="row flex-row" style="justify-content: flex-end;">
-      <img src="/images/pages/league/text_dont_just_play_code.svg" class="img-responsive" style="max-width: 410px; margin: 0 0 0 auto;" />
+    <div class="row flex-row text_dont_just_play_code" style="justify-content: flex-end;">
```

Comment:
Nitpick: Rationale for mixing `-` and `_` as separator for class name?

`text-dont-just-play-code` stays consistent with class naming.